### PR TITLE
Fix mac build to include ogg+

### PR DESCRIPTION
**Pull request by tchagnon**
Fix bad dylib in #4 that doesn't include ogg/flac, etc.  Addresses #3 .

### CODE REVIEW

**Comment by mgeier** [comment]
Thanks for the update!

I've tried pinging the ogg project about a new release https://github.com/xiph/ogg/issues/71.

Did you try if OPUS also works?

**Comment by tchagnon** [comment]
I didn't try OPUS.  I just ran the `python setup.py test` suite from SoundFile, locally pointing to this branch.

File: README.md [hunk 1]
```diff
@@ -38,16 +38,7 @@ The XCode CLI tools were installed with:
...
-
-    OGGNAME=libogg-1.3.4
-    VORBISNAME=libvorbis-1.3.7
-    FLACNAME=flac-1.3.3
-    OPUSNAME=opus-1.3.1
-    SNDFILE_VERSION=1.0.30
-
-
-Run the script to build `libsndfile.dylib`
+Edit the versions of required libraries in the `mac_build.sh` file, then run the script to build `libsndfile.dylib`:
```
Comment (mgeier) [inline README.md:41]: The rest of the README isn't really written like a "howto", but rather like "how it has been done", so the wording here is a bit inconsistent.

It's not a big deal, but if you like you could change it.

**Comment by mgeier** [comment]
> I didn't try OPUS. I just ran the `python setup.py test` suite from SoundFile, locally pointing to this branch.

I don't think any of the tests test any of the external formats.

It would be great if you could do some manual tests.

But at least the size of the `.dylib` looks better now, and it contains many more strings containing `flac` and `opus` etc.

**Comment by bastibe** [comment]
Outdated.

**Pull request closed without merging**